src/rdkafka_conf.h

```diff
@@ -159,7 +159,7 @@ typedef enum {
 
 /* Increase in steps of 64 as needed.
  * This must be larger than sizeof(rd_kafka_[topic_]conf_t) */
-#define RD_KAFKA_CONF_PROPS_IDX_MAX (64 * 30)
+#define RD_KAFKA_CONF_PROPS_IDX_MAX (64 * 31)
```

Comment:
Here is trying to avoid the ```rdkafka_conf.c:2175: rd_kafka_conf_new: Assertion `(64 * 30) > sizeof(*conf) && *"Increase RD_KAFKA_CONF_PROPS_IDX_MAX"' failed.``` failure.